Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Description: relax uuid dependency to what is in debian

At the moment, uuid is only at version 0.8 in debian unstable.

Upstream's move to the newer uuid involved no changes to how the code
uses uuid: https://github.com/rusqlite/rusqlite/pull/1167/files

So it should work fine with 0.8.

When we get ready to upload uuid 1.x to unstable, we should drop this
patch in debian.


--- a/Cargo.toml
+++ b/Cargo.toml
@@ -119,7 +119,7 @@
 optional = true
 
 [dependencies.uuid]
-version = "1.0"
+version = "0.8"
 optional = true
 
 [dev-dependencies.bencher]
@@ -141,7 +141,7 @@
 version = "2.6.0"
 
 [dev-dependencies.uuid]
-version = "1.0"
+version = "0.8"
 features = ["v4"]
 
 [features]
